cmd/lambda/accounts/main_test.go

```diff
@@ -1,11 +1,9 @@
 package main
 
 import (
-	"encoding/json"
 	"os"
 	"testing"
 
-	"github.com/Optum/dce/pkg/api/response"
 	"github.com/aws/aws-lambda-go/events"
 )
 
```

Comment:
Can you please add tests to run through the main's handler and init methods to call the create?

Reply:
Done